openedx/features/pakx/lms/overrides/models.py

```diff
@@ -4,18 +4,18 @@
 from django.db import models
 from opaque_keys.edx.django.models import CourseKeyField
 
-# Create your models here.
-
 
 class CourseProgressStats(models.Model):
     """
     Model to store the records of stats related to course progress and emails to learners
     """
-
+    NO_EMAIL_SENT = 0
+    REMINDER_SENT = 1
+    COURSE_COMPLETED = 2
     REMINDER_STATES = (
-        (0, 'No email sent'),
-        (1, 'Reminder email sent'),
-        (2, 'Course completion email sent')
+        (NO_EMAIL_SENT, 'No email sent'),
```

Comment:
shouldn't we update datatype for `email_reminder_status` as well and its default value as well?